tox.ini

```diff
@@ -139,29 +136,29 @@ envlist =
     {py3.9,py3.11,py3.12}-httpx-latest
 
     # Huey
-    {py2.7,py3.5,py3.11,py3.12}-huey-v{2.0}
-    {py3.5,py3.11,py3.12}-huey-latest
+    {py3.6,py3.11,py3.12}-huey-v{2.0}
+    {py3.6,py3.11,py3.12}-huey-latest
 
     # Loguru
-    {py3.5,py3.11,py3.12}-loguru-v{0.5}
-    {py3.5,py3.11,py3.12}-loguru-latest
+    {py3.6,py3.11,py3.12}-loguru-v{0.5}
+    {py3.6,py3.11,py3.12}-loguru-latest
 
     # OpenTelemetry (OTel)
     {py3.7,py3.9,py3.11,py3.12}-opentelemetry
 
     # pure_eval
-    {py3.5,py3.11,py3.12}-pure_eval
+    {py3.11,py3.12}-pure_eval
```

Comment:
3.6?

Reply:
Missed this one, will add

Reply:
Changed to `{py3.6,py3.11,py3.12}-pure_eval`